Formula/rrdtool.rb

```diff
@@ -3,7 +3,7 @@ class Rrdtool < Formula
   homepage "https://oss.oetiker.ch/rrdtool/index.en.html"
   url "https://github.com/oetiker/rrdtool-1.x/releases/download/v1.7.2/rrdtool-1.7.2.tar.gz"
   sha256 "a199faeb7eff7cafc46fac253e682d833d08932f3db93a550a4a5af180ca58db"
-  license "GPL-2.0"
+  license "GPL-2.0-or-later" => { with: "DigiRule-FOSS-exception" }
```

Comment:
```suggestion
  license "GPL-2.0-only" => { with: "DigiRule-FOSS-exception" }
```

Were did you see `-or-later`? I just skimmed through but I saw a bunch of `-only` headers:

[`rrd_utils.c`](https://github.com/oetiker/rrdtool-1.x/blob/7d84db9ebc3b96f0cf348c4cc02c4b40b2ade799/src/rrd_utils.c#L6-L8):

```c
/**
 * This program is free software; you can redistribute it and/or modify it
 * under the terms of the GNU General Public License as published by the
 * Free Software Foundation; only version 2 of the License is applicable.
 **/
```

Reply:
I attached the wrong file; I meant to refer to COPYRIGHT and not LICENSE:

[COPYRIGHT](https://github.com/Homebrew/homebrew-core/files/6116917/COPYRIGHT.txt)

Reply:
Ah, okay. I'm still a little concerned because some source files say `only version 2` but I don't know which takes precedence.

In terms of the exception, it's definitely close but since it's not identical I'm not whether we should use it. I don't feel comfortable approving just because I'm not sure, so it's probably best to get other maintainer (maybe PLC) opinions, here.

Reply:
We could just ask upstream right?

Reply:
Yup, will get on that later.